README.md

```diff
@@ -66,6 +66,10 @@ list of currently unimplemented issues that we are working on
 for the next release.
 
 ## Release Notes
+### Release Notes for 2.18.0
+* Use a multiple-tread process to index objects
```

Comment:
Can you add a note "This improves indexing performance" etc etc? Do we have an idea of how much faster indexing is? We also might add a note directing users to metacat.properties to check out the new config options.

Reply:
I will add it.